### PR TITLE
SW-12767 - Transfered PhpStorm and Codestyling

### PR DESCRIPTION
**Pull request by janbuecker**

### CODE REVIEW

**Comment by bcremer** [comment]
@janbuecker I am sorry to ask you for this but please remove all the text regarding coding style.
The information from the old wiki are outdated. We are using the established standards PSR-1 and PSR-2. 
Please simply link to both of this standards.

Please see https://github.com/shopware/shopware/blob/5.0/CONTRIBUTING.md#coding-standards 

# Coding standards
All contributions should follow the [PSR-1](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md) and [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md) coding
standards.

**Comment by bcremer** [comment]
Also all screenshots contain german captions.

**Comment by bcremer** [comment]
Some of the PHPStorm parts are duplicated here: https://developers.shopware.com/developers-guide/vagrant-phpstorm/

File: source/developers-guide/best-practise/codestyle-guide/index.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
+---
+layout: default
+title: Shopware Codestyle Guide
+github_link: developers-guide/codestyle-guide/index.md
+tags:
+  - api
+indexed: true
+---
+
+Our coding standard is based on the standards of Typo3 and Zend-Framework. A quick summary of the guide can be downloaded [here](http://community.shopware.com/files/downloads/swagcodingstandard-661541.pdf).
```
Comment (tiagojsag) [inline source/developers-guide/best-practise/codestyle-guide/index.md]: Code standards are PSR-1 and PSR-2: https://github.com/shopware/shopware/blob/5.1/CONTRIBUTING.md

Remove the link to the file

**Comment by janbuecker** [comment]
The changes will be made in another pull request since they will move into the phpstorm & vagrant guide.

**Comment by janbuecker** [comment]
Will be handled in #172.

**Pull request closed without merging**